django/thunderstore/webhooks/models/audit.py

```diff
@@ -20,3 +30,46 @@ class AuditWebhook(TimestampMixin):
 
     def __str__(self):
         return self.name
+
+    @classmethod
+    def get_for_event(cls, event: AuditEvent) -> QuerySet["AuditWebhook"]:
+        return cls.objects.filter(
+            Q(is_active=True)
+            & Q(Q(match_communities=None) | Q(match_communities=event.community_id))
+        )
+
+    @staticmethod
+    def get_event_color(action: AuditAction) -> int:
+        if action == AuditAction.PACKAGE_APPROVED:
+            return 5763719
+        if action == AuditAction.PACKAGE_REJECTED:
+            return 15548997
+        return 9807270
+
+    @staticmethod
+    def render_event(event: AuditEvent) -> DiscordPayload:
+        agent = User.objects.prefetch_related("social_auth").get(pk=event.user_id)
```

Comment:
The information could be accessed directly through the social auth model instead of User, saving a join operation?

Reply:
Pretty sure it's this way around because I didn't remember what the model was called and didn't feel like making it a distraction looking that up. I do agree though, it'd make the function much cleaner too